packages/ui/Forms/Error/styles.scss

```
@@ -7,9 +7,9 @@
   flex-direction: column;
   align-content: center;
   padding-top: 5px;
-  color: $theme7;
+  color: $theme5;
   @include font-size(sm);
   font-weight: 400;
-  border-top: 1px solid $theme7;
+  border-top: 1px solid $theme5;
   margin-top: -11px;
```

```suggestion
  margin-top: -10px;
```
but in general this is wrong...